### PR TITLE
[4.0] Implement the workflow into the blog sample data

### PR DESCRIPTION
**Pull request by bembelimen**
Pull Request for Issue #21707 . (first part)

### Summary of Changes
This PR let the blog sample data creating a new Workflow to ensure, that the articles are published

### Testing Instructions
Apply the "Blog Sample data" and check the workflows, a new workflow called "Sample Workflow" is created and assigned to all new sample data categories.

### CODE REVIEW

File: plugins/sampledata/blog/blog.php [hunk 1]
```diff
@@ -110,6 +111,34 @@ public function onAjaxSampledataApplyStep1()
 		$language   = Multilanguage::isEnabled() ? Factory::getLanguage()->getTag() : '*';
 		$langSuffix = ($language !== '*') ? ' (' . $language . ')' : '';
 
+		// Create a sample workflow
+		$workflowModel = new WorkflowModel;
+
+		$workflow = [
+			'title'			=> Text::_('PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_WORKFLOW_0_TITLE'),
+			'description'	=> Text::_('PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_WORKFLOW_0_DESCRIPTION'),
```
Comment (infograf768) [inline plugins/sampledata/blog/blog.php]: align =>

File: administrator/language/en-GB/en-GB.plg_sampledata_blog.ini [hunk 2]
```diff
@@ -26,6 +26,8 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_5_INTROTEXT="<p>Templates control
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_5_TITLE="Your Template"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_CATEGORY_0_TITLE="Blog"
 PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_CATEGORY_1_TITLE="Help"
+PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_WORKFLOW_0_DESCRIPTION="A sample workflow for the blog sample datas"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_sampledata_blog.ini]: ==> data

**Comment by brianteeman** [comment]
Articles, categories and menus created but no workflow

image

image

**Comment by brianteeman** [comment]
Removed my last comment as it was on the wrong branch

**Comment by brianteeman** [comment]
Assuming that the new workflow will get the full set of stages and workflows when the other pr is merged this is working correctly. If you can confirm that assumption I can mark a successful test

**Comment by bembelimen** [comment]
Yes, that's true

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 13450bc12eca8ae01e16ed605cb68563fa64bf16<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21953">issues.joomla.org/tracker/joomla-cms/21953</a>.</sub>

**Comment by infograf768** [comment]
Sorry, which other PR? I am lost here.

**Comment by bembelimen** [comment]
https://github.com/joomla/joomla-cms/pull/21963 @infograf768

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 13450bc12eca8ae01e16ed605cb68563fa64bf16<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21953">issues.joomla.org/tracker/joomla-cms/21953</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Pull request merged**